src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -382,12 +382,10 @@ class npc_eye_of_acherus : public CreatureScript
 
                 me->SetReactState(REACT_PASSIVE);
 
-                me->GetMotionMaster()->MovePoint(POINT_EYE_FALL, EyeOFAcherusFallPoint, false);
-
                 Movement::MoveSplineInit init(me);
                 init.MoveTo(EyeOFAcherusFallPoint.GetPositionX(), EyeOFAcherusFallPoint.GetPositionY(), EyeOFAcherusFallPoint.GetPositionZ(), false);
                 init.SetFall();
-                init.Launch();
+                me->GetMotionMaster()->LaunchMoveSpline(std::move(init), POINT_EYE_FALL, MOTION_SLOT_ACTIVE);
```

Comment:
@Shauren before they had to use movepoint and spline to have movementinform + "custom args"

Reply:
there are only two cases of this though

Reply:
Doesn't `MovementInform` hook need to be updated as well to check EFFECT_MOTION_TYPE insted of POINT_MOTION_TYPE?

Reply:
true true